pkgs/tools/misc/cloc/default.nix

```diff
@@ -1,31 +1,29 @@
-{ stdenv, fetchurl, fetchpatch, perl, AlgorithmDiff, RegexpCommon }:
+{ stdenv, fetchFromGitHub, makeWrapper, perl, AlgorithmDiff, RegexpCommon }:
 
 stdenv.mkDerivation rec {
-
   name = "cloc-${version}";
+  version = "1.66";
 
-  version = "1.64";
-
-  src = fetchurl {
-    url = "mirror://sourceforge/cloc/cloc-${version}.tar.gz";
-    sha256 = "1w3mz69h2i7pscvi9q7yp7wimds8g38c5ph78cj5pvjl5wa035rh";
+  src = fetchFromGitHub {
+    owner = "AlDanial";
+    repo = "cloc";
+    rev = "v${version}";
+    sha256 = "1xj6d8x8zwijym5wznm0k1478z7zx3bfjsx20r3aqb1vhkvcjsm9";
   };
 
-  patches = [ (fetchpatch {
-    name = "perl-5.22.patch";
-    url = "https://bugs.archlinux.org/task/45494?getfile=13174";
-    sha256 = "1xxwqjy2q2fdza7kfp9ld0yzljkdsrgm8a9pwnmx5q4adigcjjsz";
-  }) ];
+  sourceRoot = "cloc-v${version}-src/Unix";
 
-  buildInputs = [ perl AlgorithmDiff RegexpCommon ];
+  buildInputs = [ makeWrapper perl AlgorithmDiff RegexpCommon ];
 
   makeFlags = [ "prefix=" "DESTDIR=$(out)" "INSTALL=install" ];
 
+  postFixup = "wrapProgram $out/bin/cloc --prefix PERL5LIB : $PERL5LIB";
+
   meta = {
     description = "A program that counts lines of source code";
     homepage = http://cloc.sourceforge.net;
```

Comment:
Good point. I will force push a new commit with the homepage set to the github project page.